app/api_alpha/views.py

```diff
@@ -271,6 +274,134 @@ def get(self, request, *args, **kwargs):
             return Response(query_serializer.errors, status=400)
 
 
+class BuildingAddressView(RNBLoggingMixin, APIView):
+    @rnb_doc(
+        {
+            "get": {
+                "summary": "Identification de bâtiments par leur adresse",
+                "description": "Ce endpoint permet d'obtenir une liste paginée des bâtiments associés à une adresse. NB : l'URL se termine nécessairement par un slash (/).",
+                "operationId": "address",
+                "parameters": [
+                    {
+                        "name": "q",
+                        "in": "query",
+                        "description": "Liste les bâtiments du RNB associés à cette adresse. L'adresse fournie est recherchée dans la BAN afin de récupérer la clé d'interopérabilité associée. C'est via cette clé que sont filtrés les bâtiments. Si le geocodage échoue aucun résultat n'est renvoyé et le champ 'status' de la réponse contient 'geocoding_no_result'",
```

Comment:
Ca me permet de commencer la description de ce paramètre en disant que c'est une "liste de bâtiments du rnb ..." alors qu'il s'agit d'une adresse texte.


```suggestion
                        "description": "Adresse texte non structurée. L'adresse fournie est recherchée dans la BAN afin de récupérer la clé d'interopérabilité associée. C'est via cette clé que sont filtrés les bâtiments. Si le geocodage échoue aucun résultat n'est renvoyé et le champ <code>status</code> de la réponse contient <code>geocoding_no_result</code>",
```

Reply:
Tu mets les balises <code> pour une raison particulière ? Je vois qu'ils n'apparaissent pas dans la doc sur le gitbook, ils sont supprimés quelque part en route on dirait.

Reply:
Le changement de la première phrase est pertinent 👍